lib/geocoder.js

```diff
@@ -0,0 +1,12 @@
+// This module was originally intended to do the geocoding itself
+// BUT I couldn't get the callback to return for server.js response
+// Heroku Site API Public Key via th.good@gmail.com account
+// pubGoogKy = "AIzaSyC2rmfA9qFAXBLtPm9Ls12eapMlE-fe2no";
```

Comment:
Usually, you don't want to put your API keys on GitHub. This is where you'd use environmental variables; you'd set an environmental variable equal to your API key, and then reference that env. variable in your code. Heroku also lets you add API keys for deployment, so your code doesn't have to change.